### PR TITLE
migrate from lazy_static to once_cell across project

### PR DESCRIPTION
**Pull request by mcginty**
Carrying over a suggestion from #258 to the whole codebase - it looks like `std` will eventually integrate something like this crate, and might as well not have two of basically the same dependency in the mean time.

(Also got rid of a compiler warning about an unneeded `mut`.)

### CODE REVIEW

File: client/Cargo.toml [hunk 1]
```diff
@@ -32,6 +31,7 @@ ureq = { version = "2", default-features = false, features = ["json"] }
 wireguard-control = { path = "../wireguard-control" }
 
 [dev-dependencies]
+lazy_static = "1"
```
Comment (strohel) [inline client/Cargo.toml:34]: There's still one last usage of `lazy_static!` in `data_store.rs` tests - is that intentional?

**Review by strohel — APPROVED** [review]
Thanks, this is much more ergonomic. Just one last question.

File: client/Cargo.toml (hunk 1 above)
Comment (mcginty) [reply client/Cargo.toml]: Ok, updated :).

Comment (strohel) [reply client/Cargo.toml]: Hehe, you're quick, thanks. .)

**Pull request merged**